### ISSUE TITLE
OSSU course validation

### ISSUE THREAD

**Issue by pierrebeaucamp** [open-source-society/help#3]
Hi,

First off: I love what you're building here and I'm enrolled in the CS course myself. I think OSSU has lots of potentials.

My question is how you're planning on validating the completion of online courses? In your FAQ, you're speaking of building a portfolio as part of showcasing students accomplishments. But so far, student progress is only measured in their respective comments in the *Students Profile* issues. This seems a bit overly simplistic and insecure.

Both [edX](https://open.edx.org/open-edx-rest-apis) and [Coursera](https://tech.coursera.org/app-platform/) have APIs which could be leveraged to get the progress of courses automatically and more securely. 

Having the ability to securely certify the accomplishment of students is (in my opinion) a key milestone in becoming an accredited educational institution.

**Comment by ericdouglas** [comment]
Hey @pierrebeaucamp !

Thank you so much for your feedback!

We're developing a system that'll allow our students to track their progress.

But in general, our proposal is that our students should focus on creating projects. In that way, courses' status is not too relevant for us.

You can see both backend and frontend applications [here](https://github.com/open-source-society).

:smile:

**Comment by pierrebeaucamp** [comment]
I know about the front & backend repos but I didn't see any integration with the edX / Coursera APIs so far. 

Do you have any documents stating the roadmap for OSSU? Right now, it's a bit hard to follow the development of you guys, as there are next to no Github issues and your membership request form is closed.

**Comment by ericdouglas** [comment]
@pierrebeaucamp 

the frontend and backend teams are using [HansoftX](https://hansoftx.com) to control the upcoming releases.

We'll work on a public file to keep all the community informed about the next releases :smile:

Thank you for your interest!
